test/src/specs/sync/ibd_process.rs

```diff
@@ -25,21 +25,60 @@ impl Spec for IBDProcess {
         node0.connect(node2);
         node0.connect(node3);
         node0.connect(node4);
+        // The node's outbound connection does not retain the peer which in the ibd state
+        node0.generate_blocks(1);
         // will never connect
         node0.connect_uncheck(node5);
         node0.connect_uncheck(node6);
-        node0.generate_blocks(1);
 
         sleep(Duration::from_secs(5));
 
-        let rpc_client = node0.rpc_client();
-        let ret = wait_until(10, || {
-            let peers = rpc_client.get_peers();
+        let rpc_client0 = node0.rpc_client();
+        let is_connect_peer_num_eq_4 = wait_until(10, || {
+            let peers = rpc_client0.get_peers();
             peers.len() == 4
         });
 
-        if !ret {
+        if !is_connect_peer_num_eq_4 {
             panic!("refuse to connect fail");
         }
+
+        // IBD only with outbound/whitelist node
+        let rpc_client1 = node1.rpc_client();
+        let rpc_client2 = node2.rpc_client();
+        let rpc_client3 = node3.rpc_client();
+        let rpc_client4 = node4.rpc_client();
+        let rpc_client5 = node5.rpc_client();
+        let rpc_client6 = node6.rpc_client();
+
+        #[allow(clippy::nonminimal_bool)]
```

Comment:
Why clippy reports this?

Reply:
I tried locally, it seems that we don't need this.

Reply:
emmm, maybe I have an illusion. . why don't clippy report warning now?